src/core_plugins/kibana/public/management/sections/settings/components/field/__snapshots__/field.test.js.snap

```diff
@@ -195,10 +195,12 @@ exports[`Field for array setting should render user value if there is user value
               grow={true}
               size="xs"
             >
-              Default: 
-              <EuiCode>
-                default_value
-              </EuiCode>
+              <UNDEFINED>
```

Comment:
`UNDEFINED` in place of `React.Fragment` is a bug which is fixed with jest >= v23.0.0. I'm investigating a possible jest upgrade in another branch.